cranelift/README.md

```diff
@@ -30,30 +30,54 @@ Status
-
-The core codegen crates have minimal dependencies, support no\_std mode
-(see below), and do not require any host floating-point support, and
-do not use callstack recursion.
-
-Cranelift does not yet perform mitigations for Spectre or related
-security issues, though it may do so in the future. It does not
-currently make any security-relevant instruction timing guarantees. It
-has seen a fair amount of testing and fuzzing, although more work is
-needed before it would be ready for a production use case.
-
-Cranelift's APIs are not yet stable.
-
-Cranelift currently requires Rust 1.37 or later to build.
+WebAssembly (MVP and various extensions like SIMD), although it needs to be
+used within an external WebAssembly embedding such as Wasmtime to be part of a
+complete WebAssembly implementation. It is also usable as a backend for
+non-WebAssembly use cases: for example, there is an effort to build a [Rust
+compiler backend] using Cranelift.
+
+Cranelift is production-ready, and is used in production in several places, all
+within the context of Wasmtime. It is carefully fuzzed as part of Wasmtime with
+differential comparison against V8 and the executable Wasm spec, and the
+register allocator is separately fuzzed with symbolic verification. There is an
+active effort to formally verify Cranelift's instruction-selection backends. We
+take security seriously and have a [security policy] as a part of Bytecode
+Alliance.
+
+Cranelift has three backends: x86-64, aarch64 (aka ARM64), and s390x (aka IBM
+Z). All three backends fully support enough functionality for Wasm MVP, and
+x86-64 and aarch64 fully support SIMD as well. On x86-64, Cranelift supports
+both the System V AMD64 ABI calling convention used on many platforms and the
+Windows x64 calling convention. On aarch64, Cranelift supports the standard
+Linux calling convention and also has specific support for macOS (i.e., M1 /
+Apple Silicon).
+
+Cranelift's code quality is within range of competitiveness to browser JIT
+engines' optimizing tiers. A [recent paper] includes third-party benchmarks of
+Cranelift, driven by Wasmtime, against V8 and an LLVM-based Wasm engine, WAVM
+(Fig 22).  The speed of Cranelift's generated code is ~2% slower than that of
+V8 (TurboFan), and ~14% slower than WAVM (LLVM). Its compilation speed, in the
+same paper, is measured as approximately an order of magnitude faster than WAVM
+(LLVM). We continue to work to improve both measures.
+
+[Rust compiler backend]: https://github.com/bjorn3/rustc_codegen_cranelift
+[security policy]: https://bytecodealliance.org/security
+[recent paper]: https://arxiv.org/abs/2011.13127
+
+The core codegen crates have minimal dependencies and are carefully written to
+handle malicious or arbitrary compiler input: in particular, they do not use
+callstack recursion.
+
+Cranelift performs some basic mitigations for Spectre attacks on heap bounds
+checks, table bounds checks, and indirect branch bounds checks; see #1032 for
```

Comment:
`#1032` won't automatically become a link here, so it needs to be explicitly linked.

Reply:
Fixed, thanks!